app/build.gradle

```
@@ -24,16 +24,24 @@ android {
     buildFeatures {
         dataBinding true
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
 }
 
 dependencies {
+    implementation "androidx.room:room-runtime:2.2.6"
+    annotationProcessor "androidx.room:room-compiler:2.2.6"
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
     implementation 'com.google.android.material:material:1.3.0'
     implementation 'androidx.viewpager2:viewpager2:1.0.0'
     implementation 'androidx.fragment:fragment:1.3.2'
     implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation "androidx.room:room-rxjava2:2.2.6"
+    implementation "io.reactivex.rxjava2:rxandroid:2.1.1"
```

Los ordené un poco y les puse las comillas simples nomas:

```suggestion
    implementation 'androidx.appcompat:appcompat:1.2.0'
    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
    implementation 'androidx.viewpager2:viewpager2:1.0.0'
    implementation 'androidx.fragment:fragment:1.3.2'
    implementation 'androidx.appcompat:appcompat:1.2.0'
    implementation 'androidx.room:room-rxjava2:2.2.6'
    implementation 'androidx.room:room-runtime:2.2.6'
    implementation 'com.google.android.material:material:1.3.0'
    implementation fileTree(dir: "libs", include: ["*.jar"])
    implementation 'io.reactivex.rxjava2:rxandroid:2.1.1'
    annotationProcessor 'androidx.room:room-compiler:2.2.6'
```